tests/support/__init__.py

```diff
@@ -3,7 +3,6 @@
 #
 # --- BEGIN_HEADER ---
 #
-# support - helper functions for unit testing
```

Comment:
Nit-pick: these should stay to fit existing template. I've left them in when merging.